git/git-flow.md

```diff
@@ -0,0 +1,58 @@
+# The GIT flow
+
+![GIT flow](attachments/git-flow-aanzee)
```

Comment:
The path to the image isn't working. And I don't think we need to have a second resource for our Git flow. Not sure if I like our GitHub or Confluence for this part.